### PR TITLE
fix: styling of landing page carousels

### PR DESCRIPTION
**Pull request by melniiv**
## Description
-Fixed margins of the carousel title
-Fixed spacing with and without dots
-Dots sizes
-Font sizes and linehights inside cards
-Fixed mobile view prev next buttons
-Fixed mobile view bug with categories
- Fixed outlines of the cards in carousels

### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Review by nikomakela — COMMENTED** [review]
The changes looks looks good, but there are build-time issues

File: src/domain/event/eventDetails/EventDetails.tsx [hunk 1]
```diff
@@ -53,7 +40,18 @@ const EventDetails: React.FC<Props> = (props) => {
...
       )}
-
+      {location && (
+        <div className={styles.infoRow}>
+          <div className={styles.withIcon}>
+            <IconLocation aria-hidden />
+            <LocationText
+              event={event}
+              showDistrict={false}
+              showLocationName={true}
```
Comment (nenonja) [inline src/domain/event/eventDetails/EventDetails.tsx:50]: `showLocationName` without `={true}` is shorter

**Comment by nenonja** [comment]
Let's wait to get the review environment deployed, to compare the styles against testing environemnt

File: src/domain/event/eventDetails/EventDetails.tsx (hunk 1 above)
Comment (melniiv) [reply src/domain/event/eventDetails/EventDetails.tsx:50]: i ll fix, it was a copy from previous filter, havent noticed

**Review by nenonja — APPROVED** [review]
Style changes look good. Another thing I noticed that if you open an event from carousel and then click the back button (from the card not the browser) it goes to search page when it should go back to frontpage (and scroll back to carousel prefereably)
Screenshot 2022-09-05 at 11 17 59 AM
It's not related to this pr but could have another ticket

**Comment by melniiv** [comment]
i think its a configured back url from cms.. i think, but we can check it

**Pull request merged**